oak_functions_launcher/src/lookup.rs

```diff
@@ -14,23 +14,66 @@
 // limitations under the License.
 //
 
-use crate::schema;
+use crate::{
+    channel::ConnectorHandle,
+    schema::{self, OakFunctionsAsyncClient},
+};
 use anyhow::{anyhow, Context};
 use hashbrown::HashMap;
 use prost::Message;
-use std::fs;
+use std::{fs, path::PathBuf};
+// TODO(#3680): Use once we updated dependencies.
+// use ubyte::{self, ByteUnit};
+
+// Loads lookup data from the given path, encodes it, and sends it to the client.
+pub async fn update_lookup_data(
+    client: &mut OakFunctionsAsyncClient<ConnectorHandle>,
+    lookup_data_path: &PathBuf,
+) -> anyhow::Result<()> {
+    // Fix the maximum size of a chunk to the proto limit size.
+    let max_chunk_size = 2 * (2 << 30); // ByteUnit::Gibibyte(2);
```

Comment:
This is 4GiB. Should probably be 2 * (1 << 30).

Reply:
Thank you! (Just to illustrate why I need to `use ubyte::{self, ByteUnit};`)